packages/tools/u-boot-tools/package.mk

```diff
@@ -16,8 +16,18 @@ unpack() {
 }
 
 make_host() {
-  make qemu-x86_64_defconfig HOSTCC="${HOST_CC}" HOSTCFLAGS="-I${TOOLCHAIN}/include" HOSTLDFLAGS="${HOST_LDFLAGS}"
-  make tools-only HOSTCC="${HOST_CC}" HOSTCFLAGS="-I${TOOLCHAIN}/include" HOSTLDFLAGS="${HOST_LDFLAGS}"
+case "${MACHINE_HARDWARE_NAME}" in
+    "aarch64")
+       make tools-only_defconfig HOSTCC="${HOST_CC}" HOSTCFLAGS="-I${TOOLCHAIN}/include" HOSTLDFLAGS="${HOST_LDFLAGS}"
+       make tools-only HOSTCC="${HOST_CC}" HOSTCFLAGS="-I${TOOLCHAIN}/include" HOSTLDFLAGS="${HOST_LDFLAGS}"
+      ;;
+    "arm")
+      ;;
+    "x86_64")
+       make qemu-x86_64_defconfig HOSTCC="${HOST_CC}" HOSTCFLAGS="-I${TOOLCHAIN}/include" HOSTLDFLAGS="${HOST_LDFLAGS}"
```

Comment:
Any downside using tools-only_defconfig here too? After all, whole idea of this package is to build tools only. IIRC, tools-only_defconfig is recent addition.

Reply:
I didn’t test that combination yet (was focussed on getting aarch64 AMLGX built - just now 🎉.) But I think it can all be squashed as you say. I’ll take a `tools-only_defconfig` and test on my x86 build host, and validate.

Reply:
@jernejsk - tested build on x86 now. Minor change with lib/lib64 in nettle host build but otherwise they behave the same.